src/kixi/hecuba/minorca.clj

```diff
@@ -1,18 +1,46 @@
 (ns kixi.hecuba.minorca
   (:gen-class :main true)
-  (:require [clj-http.client         :as client]
-            [clojure.data.csv        :as csv]
-            [clojure.data.json       :as json]
+  (:require [clojure.data.csv        :as csv]
+            [clojure.edn             :as edn]
             [clojure.java.io         :as io]
             [clojure.tools.logging   :as log]
             [kixi.hecuba.api-helpers :as api]
-            [clojure.set             :as set]))
+            [clojure.set             :as set]
+            [aws.sdk.s3              :as s3]))
 
+
+;; Get all information from the configuration file
+(defn get-config
+  "Gets info from a config file."
+  [file-path]
+  (edn/read-string (slurp file-path)))
+
+(def config-info
+  (let [f (io/file (System/getProperty "user.home")
+                   ".k.h.minorca-config.edn")]
+    (get-config f)))
+
+
+;; Get all the files in the S3 bucket
+(defn list-files-in-bucket
+  "List the files contained in the s3 bucket.
+  Ordered by last modification date."
+  [name cred]
+  (log/info "Looking for the files in s3 bucket...")
+  (if (s3/bucket-exists? cred name)
+    (let [s3-objects (:objects (s3/list-objects cred name))]
```

Comment:
This is a paged call, so if you have more than one page of objects you need to do more work (same issue in amazonica).